src/main/java/org/corfudb/infrastructure/LogUnitServer.java

```diff
@@ -256,26 +253,48 @@ public void reset() {
         String d = serverContext.getDataStore().getLogDir();
         localLog.close();
         if (d != null) {
-            Path dir = FileSystems.getDefault().getPath(d);
-            String prefixes[] = new String[]{"log"};
-
-            for (String pfx : prefixes) {
-                try (DirectoryStream<Path> stream =
-                             Files.newDirectoryStream(dir, pfx + "*")) {
-                    for (Path entry : stream) {
-                        // System.out.println("Deleting " + entry);
-                        Files.delete(entry);
-                    }
-                } catch (IOException e) {
-                    log.error("reset: error deleting prefix " + pfx + ": " + e.toString());
+            Path dir1 = FileSystems.getDefault().getPath(d + "/log");
+            String report = "";
+
+            // Today I learned that the globbing done by newDirectoryStream does not
+            // handle any subdirectory "*/*" matching.  :-(
```

Comment:
These kinds of informal comments have no place in the master branch. You may have learned that newDirectoryStream does not handle subdirectory matching, but how is that useful for anyone else?

If it was really necessary for the world at large to know that DirectoryStream does not handle */* matching you would put something like:

`//Globbing by DirectoryStream does not handle subdirectory matching, so we must match using *-* instead.`